Graphics/GraphicsEngineOpenGL/src/GLContextState.cpp

```diff
@@ -88,7 +88,9 @@ void GLContextState::Invalidate()
     // Unity messes up at least VAO left in the context,
     // so unbid what we bound
     glUseProgram(0);
+#ifndef PLATFORM_EMSCRIPTEN
     glBindProgramPipeline(0);
+#endif
```

Comment:
glBindProgramPipeline may be unsupported on Android too, should be 
if (PipelineSupported)
glBindProgramPipeline();